src/main/java/com/multithread/ExecutorServiceSum.java

```diff
@@ -0,0 +1,45 @@
+package com.multithread;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class ExecutorServiceSum {
+    private int numberOfThreads;
+    private List<Integer> intList;
+
+    public Long calculateSum() {
+        int partitionListLength = intList.size() / numberOfThreads;
+        ExecutorService executor = Executors.newFixedThreadPool(numberOfThreads);
+        List<MyCallable> callableTasks = new ArrayList<>();
```

Comment:
```suggestion
        List<Callable> callableTasks = new ArrayList<>();
```
use abstraction where possible

Reply:
done